app/controllers/shell_controller.rb

```diff
@@ -1,19 +0,0 @@
-class ShellController < ApplicationController
-  # TODO: Remove these "ShellController", because they are for service worker functionality we no longer need.
```

Comment:
I think we're safely past mid-March at this point and I couldn't find any reference to this in the routes. cc @benhalpern